README.md

```diff
@@ -54,7 +54,7 @@ If you are using a `self-hosted` Windows runner, `GNU tar` and `zstd` are requir
 
 * `key` - An explicit key for a cache entry. See [creating a cache key](#creating-a-cache-key).
 * `path` - A list of files, directories, and wildcard patterns to cache and restore. See [`@actions/glob`](https://github.com/actions/toolkit/tree/main/packages/glob) for supported patterns.
-* `restore-keys` - An ordered list of prefix-matched keys to use for restoring stale cache if no cache hit occurred for key.
+* `restore-keys` - An ordered multiline string listing the prefix-matched keys, that are used for restoring stale cache if no cache hit occurred for key.
```

Comment:
Can you update https://github.com/actions/cache/blob/e47d9f9ec8be3ac771b0b03c8e50f86e1c2fdea8/restore/action.yml#L11-L13 and https://github.com/actions/cache/blob/e47d9f9ec8be3ac771b0b03c8e50f86e1c2fdea8/action.yml#L11-L13 to match this new description?

Reply:
Done @joshmgross Mind taking a relook?